hereditary/ontology/genomics/.htaccess

```diff
@@ -26,6 +26,14 @@ RewriteRule ^$ http://hereditary.dei.unipd.it/ontology/genomics/schema/hero_geno
 #RewriteCond %{HTTP_ACCEPT} text/turtle
 RewriteRule ^$ http://hereditary.dei.unipd.it/ontology/genomics/schema/hero_genomics.ttl [R=303]
 
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^schema/$ http://hereditary.dei.unipd.it/ontology/genomics/schema/hero_genomics.owl [R=303]
+
+# Rewrite rule to serve turtle content from the vocabulary URI if requested
+#RewriteCond %{HTTP_ACCEPT} text/turtle
```

Comment:
you commented out the rewrite rule. If you do this, then the ttl file will be returned for anyone accessing /schema